libs/designer-ui/src/lib/editor/base/utils/parsesegments.ts

```diff
@@ -99,12 +104,20 @@ const appendChildrenNode = (
   // if is a text node, parse for tokens
   if ($isTextNode(childNode)) {
     const textContent = childNode.getTextContent();
+    const decodedTextContent = tokensEnabled ? decodeSegmentValue(textContent) : textContent;
```

Comment:
Appears to be working for some test cases, but this one is still not working:

```
replace(replace(replace('abc','&lt;','<'),'&gt;','>'),'&quot;','"') 
```

Looking into this.

Reply:
Fixed.

![disappearing-quot-fix](https://github.com/Azure/LogicAppsUX/assets/1350074/5272df1a-26a0-4b55-b682-fa81b0b9360c)